conjur/cli.py

```diff
@@ -114,10 +160,19 @@ def run(self, *args):
         if args.verbose is True:
             args.debug = True
 
-        Cli.run_client_action(resource, args)
-
-        # Explicit exit (required for tests)
-        sys.exit(0)
+        # TODO Add tests for exception handling logic
+        try:
+            Cli.run_client_action(resource, args)
+        except FileNotFoundError as not_found_error:
+            sys.stdout.write(f"Error: No such file or directory: '{not_found_error.filename}'")
+            logging.info('Error: No such file or directory: %s', not_found_error.filename)
```

Comment:
This is really strange to me. Both of these two lines do almost the exact same things so you would get a repeated message on the output with them. Same comment applies to the items in the other `except` block.

Reply:
So what I found here is that one prints to the console and the other prints to the console when the --debug flag is added
```
$ ./pkg_bin/conjur-cli --debug variable get unknown

2020-12-01 09:26:15,827 INFO: 404 Client Error: Not Found for url: https://localhost/secrets/myorg/variable/unknown
404 Client Error: Not Found for url: https://localhost/secrets/myorg/variable/unknown
```

```
$ ./pkg_bin/conjur-cli variable get unknown
404 Client Error: Not Found for url: https://localhost/secrets/myorg/variable/unknown
```

Reply:
Why not make this a `logging.warn` or `logging.error` then?

Reply:
good point. I will update this. 
Does this comment about `logging.info` apply to all combinations of `sys.stdout.write` + `logging.info`. Or only on error?

Reply:
Possibly. You should always use the same (and only one) mechanism for displaying information to the user and logging is probably a better choice than any `sys` direct write since `logging` can have adjustable output levels.